src/event/player/PlayerDeathEvent.php

```diff
@@ -41,6 +41,8 @@ class PlayerDeathEvent extends EntityDeathEvent{
 	private $deathMessage;
 	/** @var bool */
 	private $keepInventory = false;
+	/** @var bool */
+	private $keepXp = false;
```

Comment:
You can use a native type for this. master is 7.4+ only now.

Reply:
Do I have to keep the getter and the setter anyway for code consistency?

Reply:
I'm not telling you to remove those.

Reply:
I know, I was just asking